content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions.md

```diff
@@ -628,7 +628,7 @@ Additionally, the following restrictions are applied to the schema:
 
 These fields can only be set with specific features enabled:
 
-- `default`: can be set for `apiextensions.k8s.io/v1` CustomResourceDefinitions. Defaulting is in beta since 1.16 and requires the `CustomResourceDefaulting` feature gate to be enabled (which is the case automatically for many clusters for beta features). Compare [Validation Schema Defaulting](/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/#defaulting).
+- `default`: can be set for `apiextensions.k8s.io/v1` CustomResourceDefinitions. Defaulting is in GA since 1.17 (beta since 1.16 with the `CustomResourceDefaulting` feature gate to be enabled, which is the case automatically for many clusters for beta features). Compare [Validation Schema Defaulting](/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/#defaulting).
```

Comment:
No, the suggested sentence is wrong. In GA *and* in beta it is enabled by default. In GA you cannot disable it.

Reply:
Hello @sttts, line 631 is a bit difficult to parse.
Question: 
How does `Defaulting` compare to `Validation Schema Defaulting` found in the section named Defaulting below?

Suggested rewording:
- `default`: can be set for `apiextensions.k8s.io/v1` CustomResourceDefinitions. Defaulting is in GA since v1.17. The `CustomResourceDefaulting` feature gate is enabled automatically (which is the default case for many clusters for beta features) starting with beta v1.16. Compare to [Validation ...